src/server/debug2.rs

```diff
@@ -1154,38 +1187,27 @@ fn deivde_regions_for_concurrency<ER: RaftEngine>(
     Ok(regions_groups)
 }
 
-// `key1` and `key2` should both be start_key or end_key.
-fn smaller_key<'a>(key1: &'a [u8], key2: &'a [u8], end_key: bool) -> &'a [u8] {
-    if end_key && key1.is_empty() {
-        return key2;
-    }
-    if end_key && key2.is_empty() {
-        return key1;
-    }
-    if key1 < key2 {
-        return key1;
-    }
-    key2
-}
+#[cfg(any(test, feature = "testexport"))]
+pub fn new_debugger(path: &Path) -> DebuggerImplV2<RaftLogEngine> {
```

Comment:
```suggestion
pub fn new_debugger(path: &Path) -> DebuggerImplV2<raft_log_engine::RaftLogEngine> {
```

You can use fully qualified path, so that no need `#[allow(unused_imports)]`.